TD-3404 Added latest palette colors to the theme provider

nikitharani opened the pull request:
<!-- Insert YouTrack link if relevant. This should be a clickable link -->
<!-- Pick relevant action word --->

Closes [TD-3404](https://sce.myjetbrains.com/youtrack/issue/TD-3404/Color-Palette-Light-Mode-and-Dark-Mode)

## Changes
- Added latest palette colors to the theme provider
- tests to test all the light and dark mode colors are all defined correctly
- Added mui app bar color for both light and dark mode
- Added styles to match with figma for App header component
- Changed file and image uploader dropzone background color to backgroundColor: theme.palette.background.paper to match with figma
- Changed background colour for mui data grid
- Added mui button text color when button color is primary to match with figma
- Changed background of line plot inside card
- Overrriding a color for MuiTableCell-head in dark mode
- Added background to use palette colors for the preview components

Maybe a short description of what is not included and will come in future work where appropriate.

## Dependencies
N/A

## UI/UX
@Sowbhagya-ipg could you please verify the changes.

VIRTO.TEST
https://github.com/IPG-Automotive-UK/virto/pull/1412 has deploy preview to test all the apps
<!-- Add in screen grabs / screen recordings of the feature. Tag the UI/UX designer if you require specific feedback / approval -->
<!-- If this PR solves a bug, consider including a before and after so that the reviewer can reproduce and confirm fixed -->

## Testing notes
You can test the components on storybook

<!-- Help the reviewer test your feature with some specific steps, point them towards test data and provide scripts or postman configs etc. -->

## Author checklist

Before I request a review:

<!-- Strikethrough any items that are not relevant to this PR -->

- [x] I have reviewed my own code-diff.
- [x] I have tested the changes in Docker / a deploy-preview.
- [x] I have assigned the PR to myself or an appropriate delegate.
- [x] I have added the relevant labels to the PR.
- [x] I have included appropriate tests.
- [x] I have checked that the Lint and Test workflows pass on Github.
- [x] I have populated the deploy-preview with relevant test data.
- [x] I have tagged a UI/UX designer for review if this PR includes UI/UX changes.

m4manjesh reviewed (changes requested):
General questions to @Sowbhagya-ipg 

 Are you happy with these following changes or do you have better suggestions?

1. buttons disabled state in dialog
![Screenshot 2025-03-03 162425](https://github.com/user-attachments/assets/bb298c81-c085-4923-b045-b578ecec607e)
![Screenshot 2025-03-03 162449](https://github.com/user-attachments/assets/ad7c2331-e5f7-43fa-97b3-a5217277bd3d)

2. hover behavviour of app buttons in app launcher
![Screenshot 2025-03-03 162657](https://github.com/user-attachments/assets/0e739d44-c7cd-4d4e-9c0e-6dc37aea7708)
![Screenshot 2025-03-03 162706](https://github.com/user-attachments/assets/6a03fe4d-2f8d-4f9b-8726-af0d4e6bdb59)


3. Tab Panel header
In this branch
![Screenshot 2025-03-03 162900](https://github.com/user-attachments/assets/a7d1fecd-45d2-44a0-9bb0-ad4df6baa113)

QA
![Screenshot 2025-03-03 162833](https://github.com/user-attachments/assets/b8f95f24-9e20-4522-afb6-8461a2abb6c3)

4. Ok with Clear All text color?
![Screenshot 2025-03-03 165001](https://github.com/user-attachments/assets/b82bdb4a-c413-4cc5-bc5c-4672e6079504)




@nikitharani 

1. I think the search bar should be plain
![Screenshot 2025-03-03 164024](https://github.com/user-attachments/assets/502e635f-a79b-46c9-82db-e659c16506ca)
![Screenshot 2025-03-03 164632](https://github.com/user-attachments/assets/4f8a77a9-4017-4926-b71c-80ef297aecc6)

Figma
![Screenshot 2025-03-03 164112](https://github.com/user-attachments/assets/8e90fb71-bcc6-4983-a650-08fb886d6349)

2. Datagrid footer should be primary color 
![Screenshot 2025-03-03 164300](https://github.com/user-attachments/assets/3aadefdf-ee16-479b-84f9-286c40df128f)


5. Wizard action buttons disabled state is wrong
![Screenshot 2025-03-03 165531](https://github.com/user-attachments/assets/bae9e795-2a41-4a86-99dd-7eb52ad6b6c8)

Figma
![Screenshot 2025-03-03 170615](https://github.com/user-attachments/assets/d850aebc-4dda-4d75-b36c-376ae3cbc255)


Overall new colors looks good. Left some suggestions and questions to UI. I only reviewed RUI, once the required changes made , i will test VIRTO apps.

nikitharani commented on src/ThemeProvider/ThemeProvider.tsx:
```diff
@@ -187,14 +262,25 @@ const mainTheme: ThemeOptions = {
+            root: ({ theme }) => ({
+              "&.MuiTableCell-head": {
+                backgroundColor:
+                  theme.palette.mode === "dark"
```
Moved this to dark mode as light mode uses inherit color no need to mention it in lightmode.

nikitharani commented on src/ThemeProvider/ThemeProvider.test.tsx:
```diff
@@ -112,4 +113,76 @@ describe("ThemeProvider", () => {
       });
     }
   );
+  test("should have correct colors in light mode", () => {
```
Checked with Elena and Luke in the drop-in session they suggested not to add any tests for palette colors.

nikitharani commented:
> General questions to @Sowbhagya-ipg
> 
> Are you happy with these following changes or do you have better suggestions?
> 
> 1. buttons disabled state in dialog
>    ![Screenshot 2025-03-03 162425](https://private-user-images.githubusercontent.com/56511816/418657936-bb298c81-c085-4923-b045-b578ecec607e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NTc5MzYtYmIyOThjODEtYzA4NS00OTIzLWIwNDUtYjU3OGVjZWM2MDdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwYTAwYjcyMWE5YmIxMDFlNjIwNDc3MDlmMzI0YTI0ZjFmMjAwMGJjNjhkMzc3YmMyZmFlNjk5ZjYyZDUyYjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.otgEOw_m9vwIO5_LgFamVG3dSMZeePZ0khfOvKwyRqM)
>    ![Screenshot 2025-03-03 162449](https://private-user-images.githubusercontent.com/56511816/418657965-ad7c2331-e5f7-43fa-97b3-a5217277bd3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NTc5NjUtYWQ3YzIzMzEtZTVmNy00M2ZhLTk3YjMtYTUyMTcyNzdiZDNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwMmQwOGNjOGJlNWQzNDIxZTM2MGFiYzc1NmNiM2U5NTZiYWFhNmRiYjA5MmVlY2IxYzg3ZGFjYTdjYTYyZDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.lTH6uc0MZos6N4IXU8lTw1zB2MJcEcIhWCZVS4zqX84)
> 2. hover behavviour of app buttons in app launcher
>    ![Screenshot 2025-03-03 162657](https://private-user-images.githubusercontent.com/56511816/418658275-0e739d44-c7cd-4d4e-9c0e-6dc37aea7708.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NTgyNzUtMGU3MzlkNDQtYzdjZC00ZDRlLTljMGUtNmRjMzdhZWE3NzA4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4ZmIwN2M1MWJkMzk4YTdmMGYxYjMxY2U4ZjEwYmY5MmYxOGEyYTBhOTg2MmE1YjViYzUwZmJmMDRlMWEzNGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hYO_j_hYy-R9u0ulnF_csH7eJfTG1qAENFPdKzHWzPI)
>    ![Screenshot 2025-03-03 162706](https://private-user-images.githubusercontent.com/56511816/418658293-6a03fe4d-2f8d-4f9b-8726-af0d4e6bdb59.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NTgyOTMtNmEwM2ZlNGQtMmY4ZC00ZjliLTg3MjYtYWYwZDRlNmJkYjU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyZGNmM2UxN2I1MWI4YjI0MWM2MDAwYTZmMjJkZmFiZGFjYzZiZjAxMWEzZWI5MDkxYWZjNDk2ODk1YTgxMTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.w8gS_Nyi9hdJb3cMmOpPqxGef_az3Idj2TkGL8rEidM)
> 3. Tab Panel header
>    In this branch
>    ![Screenshot 2025-03-03 162900](https://private-user-images.githubusercontent.com/56511816/418659095-a7d1fecd-45d2-44a0-9bb0-ad4df6baa113.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NTkwOTUtYTdkMWZlY2QtNDVkMi00NGEwLTliYjAtYWQ0ZGY2YmFhMTEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxNDc3MDJlNTE2MDAzODZhYzk5NTQ3YzZkNTBkZjI4NTRiYTU1NmRhNDE2MDRiYjMzY2I4MjQyYjY3NDE5MzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.roecLLD8zAyksoOLD3XJN4FP7eiAf07DYNbyVY0kvfQ)
> 
> QA ![Screenshot 2025-03-03 162833](https://private-user-images.githubusercontent.com/56511816/418659047-b8f95f24-9e20-4522-afb6-8461a2abb6c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NTkwNDctYjhmOTVmMjQtOWUyMC00NTIyLWFmYjYtODQ2MWEyYWJiNmMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzNjgxYjUwNmQyODdhNjVkNTQ3YmNjYzMyY2U5OTg5Zjk1MzgyODBjZWNjMjk5YmRlMDQ5YTRjZTNhZDc1NWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.11Mt3ZYFgivfbynvrOUozhWmXDKEuuChriPFpjvo7eE)
> 
> 4. Ok with Clear All text color?
>    ![Screenshot 2025-03-03 165001](https://private-user-images.githubusercontent.com/56511816/418665178-b82bdb4a-c413-4cc5-bc5c-4672e6079504.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NjUxNzgtYjgyYmRiNGEtYzQxMy00Y2M1LWJjNWMtNDY3MmU2MDc5NTA0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNmMTQ2ZTg4ZTNhMTIxYjU3NWVlOWQwYTY1N2VmNzQwODQ3YmEwZTE4NGZmOWM5ZDM3MGI3ZDMzYmM1NDgyMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6udVhxTtHqvYz32KBEfYdaytOkjUyLXHS_1ztVxS6fk)
> 
> @nikitharani
> 
> 1. I think the search bar should be plain
>    ![Screenshot 2025-03-03 164024](https://private-user-images.githubusercontent.com/56511816/418661259-502e635f-a79b-46c9-82db-e659c16506ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NjEyNTktNTAyZTYzNWYtYTc5Yi00NmM5LTgyZGItZTY1OWMxNjUwNmNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3MzkyZTZkM2ExNGZlMzVmMmUwNzIzYzkzN2Y5ZmYyYTdiN2UxNDUxYzZhNDQ0NTlkZDM2YTdhNDI2NjUzY2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pnz8-33Vh7REVZg4DHlMq1sETDjiluAVrbf6hvy3TyI)
>    ![Screenshot 2025-03-03 164632](https://private-user-images.githubusercontent.com/56511816/418663423-4f8a77a9-4017-4926-b71c-80ef297aecc6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NjM0MjMtNGY4YTc3YTktNDAxNy00OTI2LWI3MWMtODBlZjI5N2FlY2M2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY5MGViMDE4Y2EyMTY5NjQ3ZDEwODE2ZTQ0NDcwNDI3YjQ3MzVlMzQxMmVjNTYwZDI5NjJkNDliNmZjMTI2ZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ZWMk4ELnT8dE___UqJU4xzeGgqAi2qFxxqAe_UpxClY)
> 
> Figma ![Screenshot 2025-03-03 164112](https://private-user-images.githubusercontent.com/56511816/418661381-8e90fb71-bcc6-4983-a650-08fb886d6349.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NjEzODEtOGU5MGZiNzEtYmNjNi00OTgzLWE2NTAtMDhmYjg4NmQ2MzQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZiZDZkZDY1YzFmYTg4ZGRkYWRkYjJjZjFiZDQ1ZTVmMjUwOTVjOGY3YWE2NjI3Mzk3OTAyNzlmM2FkOTA4OWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.roxB4NBfTI1rrP8umbXrRdHGA4PpnmO01T3rOPmQx6Q)
> 
> 2. Datagrid footer should be primary color
>    ![Screenshot 2025-03-03 164300](https://private-user-images.githubusercontent.com/56511816/418662265-3aadefdf-ee16-479b-84f9-286c40df128f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NjIyNjUtM2FhZGVmZGYtZWUxNi00NzliLTg0ZjktMjg2YzQwZGYxMjhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlYTJkNGNmZDRjZGFiMzc4YTA1ZmM0ZGZiYWI2ZDA2MDA4NTg0ZmQ0YTU0Zjc3ZjViNTQ1YmZlYTlmZjU2YTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.edw_caMYzIDyABJ7XEjLDOodJYAy-Y-stajPtJQv2LE)
> 3. Wizard action buttons disabled state is wrong
>    ![Screenshot 2025-03-03 165531](https://private-user-images.githubusercontent.com/56511816/418666566-bae9e795-2a41-4a86-99dd-7eb52ad6b6c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NjY1NjYtYmFlOWU3OTUtMmE0MS00YTg2LTk5ZGQtN2ViNTJhZDZiNmM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1NmM5MmIwMTIyOTRhMTdjMzU0YmNjYThmZDVhZjJlZjQ2MDNiODBjZGE1MGJmMGI5ODM5Y2ZkNmE1NmMwODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xgaT7bcTkLy1o5hF7sBnsZH6Z5t13OodL0TXDf7IHEg)
> 
> Figma ![Screenshot 2025-03-03 170615](https://private-user-images.githubusercontent.com/56511816/418670202-d850aebc-4dda-4d75-b36c-376ae3cbc255.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDEwOTUxMTUsIm5iZiI6MTc0MTA5NDgxNSwicGF0aCI6Ii81NjUxMTgxNi80MTg2NzAyMDItZDg1MGFlYmMtNGRkYS00ZDc1LWIzNmMtMzc2YWUzY2JjMjU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDEzMjY1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjOWNhMWRiZDlkMGQ1NTVjZjMyZDNiNzBhNTY1MWIzOTI5N2I2NmZiNTAzNTZjMTMzMTU0YjJhNjEyNDU0ODkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.EbxsPyDKHtiP0IQe0zRuOxMLp_cy_xWX_FMiDt8kUas)
> 
> Overall new colors looks good. Left some suggestions and questions to UI. I only reviewed RUI, once the required changes made , i will test VIRTO apps.

1) Fixed  button text color when disabled in dark mode
2) fixed data-grid footer color
3)Search on deploy-preview VIRTO.MODEL looks like figma
![image](https://github.com/user-attachments/assets/17302e7e-8d12-41b3-bca4-385e0fe52a77)
figma:
![image](https://github.com/user-attachments/assets/54f8fa91-6ff1-4657-9a96-b481fe4d2fb6)
3) Applied all code suggestions

m4manjesh commented on src/ThemeProvider/ThemeProvider.tsx:
```diff
@@ -47,16 +47,52 @@ declare module "@mui/material/styles" {
+const primaryDarkDark = "#41607D";
+
+// secondary main light
+const secondaryLightMain = "#FFAF2C";
```
`secondaryLightMain` and `secondaryDarkMain` are same colors, please use single variable for this

nikitharani replied:
These are 2 different colors.

m4manjesh commented on src/ThemeProvider/ThemeProvider.tsx:
```diff
@@ -47,16 +47,52 @@ declare module "@mui/material/styles" {
+const secondaryLightColor = "#FFBF56";
+
+// secondary dark light
+const secondaryDarkLight = "#B27A1E";
```
same here `secondaryDarkLight` and `secondaryDarkDark`

previous comment not addressed

m4manjesh commented on src/ThemeProvider/ThemeProvider.tsx:
```diff
@@ -169,15 +218,56 @@ const mainTheme: ThemeOptions = {
+        },
+        // MuiButton: {
+        //   styleOverrides: {
+        //     root: ({ theme }) => ({
```
remove commnented code

m4manjesh reviewed (changes requested):
Why the seach dark mode background color not fixed in RUI, seach in details page(SummaryCard) component looks like follows

![Screenshot 2025-03-04 181158](https://github.com/user-attachments/assets/d91b1837-0959-42ba-808f-e29dce5ef0de)

Please install a latest pre-release in deploy-preview to test these changes

m4manjesh reviewed (changes requested):
Assign to me when its ready

m4manjesh commented:
Changes looks good, i will approve this once we gets an approval for this [PR](https://github.com/IPG-Automotive-UK/virto/pull/1412)

m4manjesh reviewed (approved):
Looks good.
